### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by YudiXie**
Update read me:
1. update outdated score format
2. add the new tutorial link
3. delete previous duplicated links to documentation and tutorials

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,9 +10,7 @@ To that end, Brain-Score operationalizes experimental data into quantitative ben
 that any model candidate following the [`BrainModel`](brainscore_vision/model_interface.py) interface can be scored on.
 
 See the [Documentation](https://brain-score.readthedocs.io) for more details 
-and the [Tutorial](https://brain-score.readthedocs.io/en/latest/modules/model_tutorial.html) 
-and [Examples](https://github.com/brain-score/candidate_models/blob/master/examples/score-model.ipynb)
-for submitting a model to Brain-Score.
+and the [Tutorial](https://brain-score.readthedocs.io/en/latest/modules/model_tutorial.html) for submitting a model to Brain-Score.
```
Comment (mschrimpf) [inline README.md:13]:
```suggestion
, e.g. for submitting a [model](https://brain-score.readthedocs.io/en/latest/modules/model_tutorial.html) or [benchmark](https://brain-score.readthedocs.io/en/latest/modules/benchmark_tutorial.html) to Brain-Score,
as well as the [Web Tutorial](https://www.brain-score.org/tutorial/), 
and [Examples](https://github.com/brain-score/vision/blob/master/examples).
```

**Review by mschrimpf — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (YudiXie) [reply README.md:13]: These links are included in the section right following this section. We can either keep these links in this section or the next section, but I recommend not both; having duplicated links in consecutive sessions could be confusing to users.

Comment (YudiXie) [reply README.md:13]: it could be also hard to maintain since if the link is changed, multiple changes need to be made to keep them up to date

Comment (mschrimpf) [reply README.md:13]: you've got the better user perspective, I'll let you make the call!

**Comment by YudiXie** [comment]
@mschrimpf This should be good to merge, but it seems that some checks have not passed.

**Comment by mschrimpf** [comment]
@deirdre-k @samwinebrake this looks like it's from the travis errors running into time limits again. Do you prefer us to hold off on merging for now until those are fixed?

**Pull request merged**